test/success/hlevels.prl

```diff
@@ -38,19 +38,15 @@ Thm PropSet(#l:lvl) : [
       [j=1 [k] (@ ($ h a (@ q i)) k)])
 ].
 
-// This proof should be easy, but it is not...
 Thm IsPropIsProp(#l:lvl) : [
   (->
    [tyA : (U #l kan)]
    (IsProp (IsProp tyA))
   )
 ] by [
   lam tyA h1 h2 =>
-  (abs i => lam a b =>
-    let foo = (PropSet #l) [`tyA, `h1];
-    unfold IsSet IsProp;
-    use foo [`a, `b, use h1 [`a, `b], use h2 [`a, `b], `i]);
-  unfold PropSet; auto
+  abs i => lam a b =>
+    `(@ ($ (PropSet #l) tyA h1 a b ($ h1 a b) ($ h2 a b)) i)
 ].
 
 Thm IsPropIsProp'(#l:lvl) : [
```

Comment:
Wait, did you see `IsPropIsProp'` below?

Reply:
Perhaps we can delete `IsPropIsProp'` now.

Reply:
oops, I mindlessly didn't see that

Reply:
Well, I'll leave both in because the first one indicates that there are some kinks to be worked out in the tactics.